source/elements/oneMKL/source/domains/lapack/lapack.rst

```diff
@@ -8,6 +8,7 @@ LAPACK Routines
 +++++++++++++++
 
 oneMKL provides a DPC++ interface to select routines from the Linear Algebra PACKage (LAPACK), as well as several LAPACK-like extension routines.
+LAPACK routines support and require column major layout of matrices.
```

Comment:
Perhaps saying "require" is enough, since if that layout is required then it must also be supported? But either way gets the message across.

Reply:
Thanks, @rachelertl , I agree. I updated the notes to only say require.